source/get-started/bootable-usb.rst

```diff
@@ -1,30 +1,19 @@
 .. _bootable-usb:
 
-Create a bootable USB drive
+Create a bootable USB drive using BalenaEtcher
```

Comment:
I'd prefer to refer to the tool as "Etcher" instead of "balenaEther".
It looks like the program/project name is still Etcher (https://github.com/balena-io/etcher) and balena is the umbrella for the projects.

Reply:
And I wouldn't bind the title of the doc to the tool being used.